### PR TITLE
Add before start hook

### PR DESCRIPTION
**Pull request by gertjvr**
This is a new PR to replace #47

### CODE REVIEW

**Comment by gertjvr** [comment]
In response to https://github.com/elastic-coders/serverless-webpack/pull/47#issuecomment-253528703 I have created an [example](https://github.com/ticketsolutionsptyltd/serverless-offline-example) of how I would use `serverless-offline` and `serverless-webpack`

**Comment by gertjvr** [comment]
any update on this PR and when it will be merged?

File: lib/cleanup.js [hunk 1]
```diff
@@ -18,6 +18,19 @@ module.exports = {
             if (err) {
               reject(err);
             } else {
+              if (this.serverless.service.package.individually) {
```
Comment (thenikso) [inline lib/cleanup.js]: could you update with master? this part should already be present from #48

Comment (gertjvr) [inline lib/cleanup.js]: Will have a look at it tomorrow.

Comment (thenikso) [inline lib/cleanup.js]: thanks! also #55 looks similar to this PR but it also has readme improvements. would you review that one as well please? 🍰

**Pull request closed without merging**